.github/workflows/provenance.yml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2022 SLSA Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: SLSA provenance generator
+
+permissions:
+  contents: read
+
+env:
+  # This repo.
+  GENERATOR_REPOSITORY: ianlewis/slsa-github-generator
+  GENERATOR_REF: slsa-gen-lib-ianlewis
+
+###################################################################
+#                                                                 #
+#            Input and output argument definitions                #
+#                                                                 #
+###################################################################
+on:
+  workflow_call:
+    inputs:
+      artifacts:
+        description: "Formatted list of artifacts of the form NAME:SHA256[|NAME:SHA256[|...]]"
+        required: true
+        type: string
+    outputs:
+      attestation-name:
+        description: "The artifact name of the signed provenance"
+        value: ${{ jobs.sign-prov.outputs.prov-name }}
```

Comment:
Yes, Will fix.

Reply:
This actually needs a bit more since I think I'll need to define the job output for `generator` and assign that the output of the `sign-prov` step. This is a remnant from removing the `sign-prov` job that was in the original slsa-go repo.